crates/red_knot_python_semantic/resources/mdtest/call/constructor.md

```diff
@@ -1,7 +1,50 @@
 # Constructor
 
+## No init or new
+
+Here we statically infer that there is no custom `__init__` or `__new__` method defined on the
+class, hence the call to the constructor is only valid with no arguments.
```

Comment:
The reason that this works at all is that we fall back to calling `__init__` on `object`, correct?

`builtins.pyi`:
```pyi
class object:
    # …
    def __init__(self) -> None: ...
```

So maybe we can rephrase this a bit?
```suggestion
Every class has `object` in it's MRO, so if no `__init__` method is provided, we fall back to
`object.__init__`, which can only be called with zero arguments:
```